openfisca_us/parameters/irs/payroll/medicare/rate/employee.yaml

```diff
@@ -0,0 +1,10 @@
+description: Employee-side Medicare FICA rate.
+values:
+  2013-01-01: 0.0145
+metadata:
+  unit: /1
+  name: employee_medicare_rate
+  label: Employee-side Medicare tax rate
+  reference:
+    title: 26 U.S. Code § 3101 - Rate of tax
```

Comment:
Missing `-` here (but I filed https://github.com/PolicyEngine/openfisca-tools/issues/43 in the spirit of automation)